package.json

```
@@ -1,21 +1,22 @@
 {
   "name": "sarafu-network",
-  "private": true,
+  "description": "Sarafu Network is the premier dApp for interacting with CAV's (Community Asset Vouchers)",
+  "homepage": "https://github.com/owner/project#readme",
   "version": "0.0.35",
   "sideEffects": false,
+  "license": "AGPL-3.0",
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=20"
```

```suggestion
    "node": ">=18"
```
Node 20 is still in beta on Vercel 🤷